storage/blob_test.go

```diff
@@ -838,9 +838,10 @@ func (s *StorageBlobSuite) TestSetContainerPermissionsSuccessfully(c *chk.C) {
 	c.Assert(cli.CreateContainer(cnt, ContainerAccessTypePrivate), chk.IsNil)
 	defer cli.deleteContainer(cnt)
 
-	perms := s.createContainerPermissions(ContainerAccessTypeBlob, 0, "", "MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTa=", true, true, true)
+	perms := ContainerPermissions{}
+	perms.createContainerPermissions(ContainerAccessTypeBlob, "GolangRocksOnAzure", time.Now(), time.Now().Add(10*time.Hour), true, true, true)
 
-	err := cli.SetContainerPermissions(cnt, perms)
+	err := cli.SetContainerPermissions(cnt, perms, 0, "")
```

Comment:
Maybe we should make this into a method family, so that if one doesn't want to provided a lease or timeout, one doesn't have to specify `0` and `""`.